pytorch_lightning/core/lightning.py

```diff
@@ -106,6 +105,7 @@ def forward(self, batch):
                 return logits
 
         """
+        warnings.warn('Model does not have implemented `forward` method.')
```

Comment:
I would rather keep this abstract so a user has to willingly replace/fill by `pass`... @williamFalcon @neggert

Reply:
@Borda that breaks convention with pytorch. let's not keep this abstract.